src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/update-alert-notification-channels.mdx

```diff
@@ -10,9 +10,12 @@ redirects:
   - /docs/alerts/new-relic-alerts-beta/managing-notification-channels/updating-alert-notification-channels
   - /docs/alerts/new-relic-alerts-beta/managing-notification-channels/update-alert-notification-channels
   - /docs/alerts/new-relic-alerts/managing-notification-channels/update-alert-notification-channels
+  - /docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/view-or-update-user-email-channels
+  - /docs/alerts/new-relic-alerts-beta/managing-notification-channels/view-or-update-user-email-channels
+  - /docs/alerts/new-relic-alerts/managing-notification-channels/view-or-update-user-email-channels
 ---
 
-Manage your channel so your alert notifications can stay consistent with your business needs.
+Manage your channels so your alert notifications can stay consistent with your business needs.
```

Comment:
Nitpick - remove "can"

Reply:
Is this also a case where we should frontload the benefit to help drive a desired behavior?  For example:
"To keep alert notifications consistent with your business needs, manage your channels."

Reply:
Good call.